### ISSUE TITLE
Write Authorizer Lambda

### ISSUE THREAD

**Issue by angiedear** [oceanbuilders/smartpod#60]
Write stock authorizer lambda that works with cognito.
5 points

**Comment by angiedear** [comment]
@godslave05 is this a duplicate issue?

**Comment by godslave05** [comment]
@angiedear 
No this is required to to be done before the session related issue

**Comment by angiedear** [comment]
@scottdear, @godslave05 will be reaching out to you on discord.

**Comment by omerll** [comment]
Scott answered all the questions!, Still in progress!

**Comment by godslave05** [comment]
@siddiqss created an authorizer for cognito

**Comment by omerll** [comment]
Still in  prgress, will be completed wthis weekend....    We need to write a new issue

**Comment by siddiqss** [comment]
On hold b/c Asad is preparing for GRE

**Comment by omerll** [comment]
Blocked on @godslave05

**Comment by omerll** [comment]
Taking time on the process of  validating the tokens.
Still in progress
will be solved for next demo 25/nov
